e2e/test/EventHubTestListener.cs

```diff
@@ -133,7 +134,7 @@ private static bool VerifyKeyValue(string checkForKey, string checkForValue, IDi
             {
                 if (checkForKey == key)
                 {
-                    if ((string)properties[checkForKey] == checkForValue)
```

Comment:
You don't want to return true for all `null` values. You should instead pass `null` into `checkForValue` for application properties